sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -521,15 +526,34 @@ object DottyPlugin extends AutoPlugin {
       scalaLibraryJar,
       dottyLibraryJar,
       compilerJar,
-      allJars
+      allJars,
+      appConfiguration.value
     )
   }
 
   // Adapted from private mkScalaInstance in sbt
   def makeScalaInstance(
```

Comment:
Does the updated sbt-dotty plugin still work with previous versions of dotty?

Reply:
Yes it does!